src/libs/ascent/runtimes/flow_filters/ascent_runtime_conduit_to_vtkm_parsing.cpp

```diff
@@ -191,6 +196,124 @@ parse_camera(const conduit::Node camera_node, vtkm::rendering::Camera &camera)
       vtkm::Float64 elevation = camera_node["elevation"].to_float64();
       camera.Elevation(elevation);
   }
+
+  // ----------------------- //
+  // Visit camera parameters //
+  // ----------------------- //
+  if(camera_node.has_child("focus"))
+  {
+      conduit::Node n;
+      camera_node["focus"].to_float64_array(n);
+      const float64 *coords = n.as_float64_ptr();
+      vtkmVec3f look_at(coords[0], coords[1], coords[2]);
+      camera.SetLookAt(look_at);
+  }
+
+  if(camera_node.has_child("view_normal") && 
+     camera_node.has_child("focus") && 
+     camera_node.has_child("view_angle") && 
+     camera_node.has_child("parallel_scale"))
+  {
+      conduit::Node view, focus;
+      camera_node["view_normal"].to_float64_array(view);
+      camera_node["focus"].to_float64_array(focus);
+
+      const conduit::float64 *v = view.value();
+      const conduit::float64 *fc = focus.value();
+
+      conduit::float64 view_angle = camera_node["view_angle"].to_float64() * (vtkm::Pi() / 360.0);
+      conduit::float64 parallel_scale = camera_node["parallel_scale"].to_float64();
+
+      // Compute camera distance using perspective projection formula
+      double distance = (parallel_scale) / std::tan(view_angle);
+
+      // Normalize viewNormal vector
+      conduit::float64 norm = std::sqrt(v[0]*v[0] + v[1]*v[1] + v[2]*v[2]);
+      conduit::float64 v_norm[3] = { v[0]/norm, v[1]/norm, v[2]/norm };
+
+      conduit::float64 pos[3];
+      for(int i = 0; i < 3; ++i)
+      {
+        pos[i] = fc[i] + v_norm[i] * distance;
+      }
+
+      vtkmVec3f position(pos[0], pos[1], pos[2]);
+      camera.SetPosition(position);
+
+      // Clipping planes require positional information to be computed
+      if(camera_node.has_child("near_plane"))
+      {
+        vtkm::Range clipping_range = camera.GetClippingRange();
+        clipping_range.Min =std::max(0.001, distance + camera_node["near_plane"].to_float64());
```

Comment:
Why do this max? Is this to avoid the near clipping range going into the negative? Can VTKm handle that case? I think a smaller number than 0.001 would be better but maybe I'm not understanding. Should this case yield an error message instead? Or at least a warning that your near plane value is being discarded?